tests/sentry/api/endpoints/internal/test_integration_proxy.py

```diff
@@ -73,37 +74,41 @@ def setUp(self):
             integration_id=self.integration.id
         ).first()
 
+        self.valid_header_kwargs = self.create_request_headers(
+            integration_id=self.org_integration.id, signature_path=self.proxy_path
+        )
+        self.valid_request = self.factory.get(self.path, **self.valid_header_kwargs)
+
+    def create_request_headers(
+        self,
+        signature_path,
+        integration_id: str | None = None,
+        request_body=b"",
+        base_url="https://example.com/api",
+    ):
         signature = encode_subnet_signature(
             secret=self.secret,
-            base_url="https://example.com/api",
-            path=self.proxy_path,
-            identifier=str(self.org_integration.id),
-            request_body=b"",
+            base_url=base_url,
+            path=signature_path,
+            identifier=str(integration_id),
+            request_body=request_body,
```

Comment:
Cutting down on the overall clutter in this test from folks copy-pasting the same header + signature generation.